mwdb/web/src/components/ShowSample.js

```diff
@@ -234,6 +234,21 @@ function SampleDetails() {
     );
 }
 
+function xorArrayBuffer(buffer) {
+    /*
+    xor buffer with key equal 255
+	https://stackoverflow.com/questions/49885795/access-uint8array-in-javascript-arraybuffer
```

Comment:
That's a bit... extreme? Four stackoveflow references for four lines of code :).

I know that SO is technically creative commons, but it doesn't look like you've copied the code verbatim from anywhere. Please let's assume this is your original work and remove the comments. Or I can rewrite it without looking anything up if you prefer ;).

Reply:
To be honest I'm not a javascript master :) I copied & changed few chunks of code, to make working solution. Couldn't find anything better in pure js. I created solution with `numjs` dependency, which is slightly faster, but I assume you rather refuse it. However here is the mentioned part of code:
```javascript
function negateBuffer(buffer) {
    /*
    https://github.com/nicolaspanel/numjs
    */
    const length = buffer.byteLength
    const buffer_nj = nj.uint8(buffer)
    const ones = nj.ones([length], 'uint8');
    return buffer_nj.negative().subtract(ones)
}
```
If you mind to write your own solution it would be great!